src/revaultd.rs

```diff
@@ -276,6 +276,41 @@ pub struct BlockchainTip {
     pub hash: BlockHash,
 }
 
+#[derive(Debug, Clone)]
+pub enum UserRole {
+    Stakeholder,
+    Manager,
+    ManagerStakeholder,
```

Comment:
Ok to change for StakeholderManager (It is the convention we use in other repositories like `aquarium`)